api.md

```diff
@@ -1874,6 +1862,44 @@ are meant to implement data retention or perform tasks that will improve query
 performance on older chunks. Each policy is assigned a scheduled job
 which will be run in the background to enforce it.
 
+## add_continuous_aggregate_policy() :community_function> [](add_continuous_aggregate_policy)
+
+Create a policy that refresh a windows in a continuous aggregate.
+
+#### Required Arguments [](add_continuous_aggregate_policy-required-arguments)
+
+|Name|Description|
+|---|---|
+| `continuous_aggregate` | (REGCLASS) The continuous aggregate to add the policy for. |
+| `start_interval` | (INTERVAL) Start of the window relative to current time to refresh. |
+| `end_interval` | (INTERVAL) End of the window relative to current time to refresh. |
+| `schedule_interval` | (INTERVAL) Interval between refresh executions. |
+
+#### Optional Arguments [](add_continuous_aggregate_policy-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `if_not_exists` | (BOOLEAN) Set to true to avoid throwing an error if the continuous aggregate_policy already exists. A notice is issued instead. Defaults to false. |
```

Comment:
```suggestion
| `if_not_exists` | (BOOLEAN) Set to true to avoid throwing an error if the continuous aggregate policy already exists. A notice is issued instead. Defaults to false. |
```